plat/imx/common/include/imx_sip_svc.h

```diff
@@ -18,6 +18,8 @@
 #define IMX_SIP_BUILDINFO_GET_COMMITHASH	0x00
 
 #define IMX_SIP_SRC			0xC2000005
+#define IMX_SIP_SRC_M4_START		0x00
+#define IMX_SIP_SRC_M4_STARTED		0x01
```

Comment:
The naming is confusing (M4 on imx8mp...). Maybe `IMX_SIP_SRC_M4_M7_START`. Or an additional definition for M7 with the corresponding code paths - ie. `#if  defined(PLAT_imx8mn) || defined(PLAT_imx8mp)`

Reply:
Good point! Would change the naming.

Reply:
Logic has been re-worked in _V2_.